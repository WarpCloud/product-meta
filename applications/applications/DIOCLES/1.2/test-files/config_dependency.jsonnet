{
  application_type: "DIOCLES",
  application_name: "diocles",
  application_version: "1.2",
  user_config: {},
}
