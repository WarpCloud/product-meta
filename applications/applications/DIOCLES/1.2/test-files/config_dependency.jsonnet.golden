{
   "Customized_Namespace": "default",
   "Develop": false,
   "Transwarp_Install_ID": "",
   "instance_list": [
      {
         "annotations": {
            "application_name": "diocles",
            "application_type": "DIOCLES"
         },
         "configs": {
            "diocles_cpu_limit": 1,
            "diocles_cpu_request": 0.10000000000000001,
            "diocles_memory_limit": 4,
            "diocles_memory_request": 1
         },
         "dependencies": [
            {
               "moduleName": "guardian-cas",
               "name": "guardian-cas"
            }
         ],
         "instance_settings": { },
         "moduleName": "diocles",
         "name": "diocles-diocles",
         "version": "1.2"
      }
   ],
   "instance_settings": { },
   "version": ""
}
